lib/cli.js

```diff
@@ -59,7 +59,16 @@ if (opts.perf.profile && process.execArgv.indexOf('--prof') === -1) {
   return;
 }
 
-var Promise = require('bluebird');
+try {
+  var Promise = require('bluebird');
+} catch (e) {
+  if (e instanceof ReferenceError) {
```

Comment:
I wrongly assumed the require was what was throwing the Reference error. Now I see it just sets the Promise and the Reference error is thrown later on. I changed it.